setup-dev.sh

```diff
@@ -5,6 +5,8 @@ version=$(lsb_release -sr)
 
 if [ "$distro" = "openSUSE" ]; then
     packages="npm nodejs"
+elif [ "$distro" = "Ubuntu" -a ${version:0:2} -ge 19 ]; then
+    packages="npm nodejs libgconf-2-4"
```

Comment:
libgconf-2-4 is spelled differently than in the next condition after this one.

Reply:
yeah no shit, that's why we need the new condition